### ISSUE TITLE
Xcode 8 & Swift 3 Support?

### ISSUE THREAD

**Issue by Scenario** [rentzsch/mogenerator#342]
Hey Folks, what's up with Swift 3 support?

Mogenerator's output no longer compiles in the Xcode 8 Betas. And Xcode's "Create NSManagedObject Subclass" feature doesn't even come close to providing what Mogenerator did. For example, there are no string constants for property names, which many devs rely on for KVC.

This leaves Core Data app developers in quite the pickle. What's the scoop?

**Comment by rentzsch** [comment]
tl;dr: Pull Requests welcome.

Other than me noticing Apple trying to Sherlock mogenerator with Xcode 8 (which is [welcomed](https://twitter.com/rentzsch/status/743866055081111552)) I haven't looked into it. I currently don't care about Swift, and rely upon the <strike>kindness</strike> Pull Requests of Strangers for mogenerator's Swift support.

I'm not surprised property name constants aren't generated by Xcode 8. I [predicted something like this](https://twitter.com/rentzsch/status/743867083079835648). See also: fetch request wrapper generation (which I love, but no one else (statistically, probably)) uses).

**Comment by rentzsch** [comment]
Landed in mogenerator 1.32

**Comment by iwllyu** [comment]
hey @rentzsch by fetch request wrapper generation do you mean the extensions for `fetchRequest()`?

You said that'll be in 1.32?

**Comment by rentzsch** [comment]
@iwllyu yup, that's what I meant.

It won't be in v1.32 or any other version unless someone sends me a passing Pull Request that adds it to the Swift side.

**Comment by iwllyu** [comment]
Fair enough, the template file looks like greek to me at the moment but maybe it won't be so bad. Great work with mogenerator btw.
